src/middlewared/middlewared/plugins/filesystem_/stat_linux.py

```diff
@@ -0,0 +1,42 @@
+import errno
+import os
+
+import psutil
+
+from middlewared.service import CallError, Service
+from middlewared.utils.path import is_child
+
+from .stat_base import FilesystemServiceBase
+
+
+class FilesystemService(Service, FilesystemServiceBase):
+
+    def statfs(self, path):
+        try:
+            st = os.statvfs(path)
```

Comment:
@anodos325 thought maybe we could use the same for FreeBSD.

Is os.statvfs available in FreeBSD just the same?

For a quick search we are only using it in UI to get free bytes in VM section.

Reply:
@william-gr yes, it is the same, we will only lose flags and fsid